<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SubEdit_UI</class>
 <widget class="QWidget" name="SubEdit_UI">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>402</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0" colspan="4">
    <widget class="QPlainTextEdit" name="subText">
     <property name="tabChangesFocus">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QToolButton" name="buttonPrev">
     <property name="text">
      <string>...</string>
     </property>
     <property name="arrowType">
      <enum>Qt::LeftArrow</enum>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QToolButton" name="buttonNext">
     <property name="text">
      <string>...</string>
     </property>
     <property name="arrowType">
      <enum>Qt::RightArrow</enum>
     </property>
    </widget>
   </item>
   <item row="1" column="2">
    <widget class="QToolButton" name="buttonApply">
     <property name="text">
      <string>...</string>
     </property>
    </widget>
   </item>
   <item row="1" column="3">
    <spacer name="horizontalSpacer">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>250</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="2" column="0" colspan="4">
    <widget class="QListView" name="sub_list"/>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
